pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -380,6 +380,14 @@ func (pe *StandardSecurityPolicyEnforcer) EnforceDeviceMountPolicy(target string
 	for _, container := range pe.Containers {
 		for _, layer := range container.Layers {
 			if deviceHash == layer {
+				if existingHash := pe.Devices[target]; existingHash != "" {
```

Comment:
We are guaranteed that hashes stored in `Devices` is always non-empty, right?

Reply:
yeah, with the way we initialize the enforcer right now this can't be nil